cmd/daemon/init.go

```diff
@@ -53,15 +60,12 @@ func buildInitCmd(parentCmd *cobra.Command) {
 			err := wallet.CheckMnemonic(*restoreOpt)
 			cmd.FatalErrorCheck(err)
 		}
-		cmd.PrintLine()
-		cmd.PrintInfoMsgf("Enter a password for wallet")
-		password := cmd.PromptPassword("Password", true)
 
-		cmd.PrintLine()
-		cmd.PrintInfoMsgBoldf("How many validators do you want to create?")
-		cmd.PrintInfoMsgf("Each node can run up to 32 validators, and each validator can hold up to 1000 staked coins.")
-		cmd.PrintInfoMsgf("You can define validators based on the amount of coins you want to stake.")
-		numValidators := cmd.PromptInputWithRange("Number of Validators", 7, 1, 32)
+		if *valNumOpt < 1 || *valNumOpt > 32 {
```

Comment:
We can't change the previous behavior.
I suggest:
1- Don't set default in flags
2- If it is not set (or zero) ask user to set it manually